Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -13,6 +13,26 @@ final class SubscriptionsViewController: BaseViewController {
 
     @IBOutlet weak var tableView: UITableView!
     @IBOutlet weak var activityViewSearching: UIActivityIndicatorView!
+    @IBOutlet weak var imageViewIcon: UIImageView!
```

Comment:
What's this `imageViewIcon`?

Reply:
I used as a reference the "AutocompleteCell" that has this code and object in the .xib. I remove?

Reply:
@micheltlutz If it's not being used, it should be removed. Make sure it doesn't crash the app because of some XIB reference please.

Reply:
Removed and not crash